tests/test_db_function.py

```diff
@@ -184,27 +184,27 @@ def test_get_comments(seed_application_records):
         "user_id": "test",
         "theme_id": theme_id,
     }
-    create_comment_for_application_sub_crit(**assessment_payload_2)
+    create_comment(**assessment_payload_2)
 
     assessment_payload_3 = {
         "application_id": application_id,
         "sub_criteria_id": sub_criteria_id,
```

Comment:
Can we make this `None` to match how the code uses `WHOLE_APPLICATION` pls?

Reply:
And `theme_id`

Reply:
Also, can we put back what was there previously and create a new test case for the this new scenario pls

Reply:
done